liquidator/liquidator.js

```diff
@@ -118,70 +112,82 @@ class Liquidator {
     // expired and disputed liquidations.
     const expiredLiquidations = this.empClient.getExpiredLiquidations();
     const disputedLiquidations = this.empClient.getDisputedLiquidations();
-    const potentialWithdrawableLiquidations = expiredLiquidations.concat(disputedLiquidations);
+    const potentialWithdrawableLiquidations = expiredLiquidations
+      .concat(disputedLiquidations)
+      .filter(liquidation => liquidation.liquidation === this.account);
```

Comment:
Shouldn't this be `liquidation.liquidator`?

Reply:
yes, it should. Nice catch.

Reply:
Weird that that wasn't caught in the tests....

Reply:
Hm well the `liquidator` property was only added in this PR so can you try to add that to the liquidator tests to catch this specific `filter`?

Reply:
If the filter *fails*, the tests should already catch it because the withdraw would never happen.

Reply:
Agreed. Perhaps `this.account` and `liquidation.liquidation` are both undefined?

Reply:
No, I just realized that another test was failing before it got to this test. I tested it and 3 of the Liquidator tests were failing. Fixing this typo made them pass. We should be good :)

Reply:
Done.